src/inject/dynamic-theme/modify-css.ts

```diff
@@ -85,9 +85,29 @@ export function getModifiedUserAgentStyle(theme: Theme, isIFrame: boolean, style
     lines.push(`    border-color: ${modifyBorderColor({r: 76, g: 76, b: 76}, theme)};`);
     lines.push(`    color: ${modifyForegroundColor({r: 0, g: 0, b: 0}, theme)};`);
     lines.push('}');
-    lines.push('a {');
-    lines.push(`    color: ${modifyForegroundColor({r: 0, g: 64, b: 255}, theme)};`);
-    lines.push('}');
+    let linkColor: string;
+    let visitedLinkColor: string;
+    if (theme.linkColor === 'auto') {
+        linkColor = modifyForegroundColor({r: 0, g: 64, b: 255}, theme);
+    } else {
+        const hsl = rgbToHSL(parse(theme.linkColor));
+        linkColor = hslToString(hsl);
+        const h: number = (hsl.h > 315 ? hsl.h - 360 : hsl.h) + 45;
+        visitedLinkColor = hslToString({
+            l: clamp(hsl.l + 0.2, 0, 1),
+            s: clamp(hsl.s - 0.3, 0, 1),
+            h});
+    }
+    if (linkColor) {
+        lines.push('a {');
+        lines.push(`    color: ${linkColor};`);
```

Comment:
I don't see any effect in most cases, maybe let's use `!important` if it's not `auto`?